composer.json

```diff
@@ -32,7 +32,8 @@
       "php-stubs/woocommerce-stubs": "5.9.0",
       "rregeer/phpunit-coverage-check": "0.3.1",
       "yoast/phpunit-polyfills": "1.0.1",
-      "cweagans/composer-patches": "1.7.1"
+      "cweagans/composer-patches": "1.7.1",
+      "automattic/jetpack-changelogger": "3.0.5"
```

Comment:
Is there a reason for specifying exact version?

Reply:
No exact reason. But this is a good point, I've changed it to `^3.0.7` to get the maximum benefit of compatibility. 4091055